source/_integrations/seventeentrack.markdown

```diff
@@ -72,3 +72,14 @@ content: >-
 ```
 
 {% endraw %}
+
+Attributes that can be using in Lovelace markdown:
```

Comment:
As asked in the previous PR as well. What attributes? How is this related to the Lovelace markdown? The use case you are documenting right now, is really specific, while it are essentially just the attributes available on the entity?